test/cli/install/bun-update.test.ts

```diff
@@ -133,7 +133,7 @@ it("should update to latest version of dependency", async () => {
   expect(await file(join(package_dir, "package.json")).json()).toEqual({
     name: "foo",
     dependencies: {
-      baz: "^0.0.5",
+      baz: "~0.0.5",
```

Comment:
Can we run this test for both `~0.0.5` and `^0.0.5`?

Reply:
👍